node_modules/gh-core/lib/db.js

```diff
@@ -353,11 +353,13 @@ var _setUpModel = function(sequelize) {
      * @property  {String}      displayName         The display name of the serie
      * @property  {String}      description         The description of the serie
      * @property  {String}      image               The path to an image for the serie
+     * @property  {Object}      metadata            The extra metadata of the serie
```

Comment:
For the timetable app it contains aggregated info for:
 * times
 * places
 * organisers

I'm not sure yet what (or if) we're going to use this field for when doing talks. I feel like `summary` doesn't really cover it either